package.json

```diff
@@ -84,10 +84,10 @@
     "jest": "^30.0.5",
     "jose": "^5.9.6",
     "markdownlint-cli": "^0.45.0",
-    "nock": "^14.0.7",
+    "nock": "^14.0.8",
     "prettier": "^3.6.2",
-    "ts-jest": "^29.4.0",
-    "typescript": "^5.8.3",
-    "typescript-eslint": "^8.38.0"
+    "ts-jest": "^29.4.1",
+    "typescript": "^5.9.2",
+    "typescript-eslint": "^8.39.0"
```

Comment:
needed to update typescript-eslint to latest version to support typescript@5.9.2